# 한글탑 테스트 비행 GPS 경로
# WPT_1: [37.5625148, 126.9366716, 67.26438290361696]
# WPT_2: [37.5625972, 126.9367649, 67.26630097491743]
# WPT_3: [37.5625561, 126.9365968, 67.29012593961525]

# 송도 GPS 경로
# WPT_1: [37.3818628, 126.6717931, 0.0]
# WPT_2: [37.3820336, 126.6720268, 37.0]
# WPT_3: [37.3820213, 126.6720402, 37.0]

# 태안 UV랜드 GPS경로
WPT_1: [36.662052, 126.341461, 15.0]
WPT_2: [36.6610271, 126.3419901, 15.0]
WPT_3: [36.6602715, 126.3424106, 15.0]

# 아식공동설계연구소
# WPT_1: [37.5721873, 126.9400619, 15.0]
# WPT_2: [37.572174, 126.9402222, 15.0]
# WPT_3: [37.5721581, 126.9403724, 15.0]
